core/metacling/src/TCling.cxx

```diff
@@ -1798,9 +1806,12 @@ void TCling::RegisterModule(const char* modulename,
       if (dyLibName && !cling::DynamicLibraryManager::isSharedLibrary(dyLibName))
          dyLibName = nullptr;
 
-      if (dyLibName) {
 
-         // We were able to determine that is a valid library.
+      if (dyLibName) {
+         // We need to open the dictionary shared library, to resolve symbols
```

Comment:
Why do we have the same comment here as in line 1797, in the new version of the file?